pkgs/development/python-modules/aws-encryption-sdk/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, wrapt
+, cryptography
+, attrs
+, boto3
+, pytestCheckHook
+, pytest-mock
+, mock
+}:
+
+buildPythonPackage rec {
+  pname = "aws-encryption-sdk";
+  version = "3.1.1";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0qwmgwzhakb8xdby97vafg3rcll0zlvx0kdymhf6p3f6iw0vypwd";
```

Comment:
Could you please use SRI hash?
> Nixpkgs contributors are currently recommended to use hash.

https://nixos.org/manual/nixpkgs/stable/#fetchurl